.changelogs/11091.json

```diff
@@ -0,0 +1,8 @@
+{
+  "issuesOrigin": "private",
+  "title": "fix aria-label, aria-checked and menuitemcheckbox role for the readOnly and alignment contextMenu items",
```

Comment:
- Start with capital letter, sentence case
- typo role > roles because you are referring to many items not just one

Reply:
Fixed